frontends/common/options.h

```diff
@@ -109,4 +109,36 @@ class CompilerOptions : public Util::Options {
     DebugHook getDebugHook() const;
 };
 
+
```

Comment:
I expect an `#include` for lib/compile_cointext.h

Reply:
Thanks; I'll fix it.